oeps/oep-0005/decisions/0001-location-of-decentralized-docs.rst

```diff
@@ -24,7 +24,8 @@ Decision
 Place DD documentation(general decisions, general how-tos) in edx/open-edx-proposals/oeps/oep-0005. Eventually, these decisions will be used to update OEP-5.
 
 .. note::
-Any documentation specific to a particular repo should still go in that repo regardless of its relation to DD.
+
+    Any documentation specific to a particular repo should still go in that repo regardless of its relation to DD.
```

Comment:
I'm not the biggest fan of using the DD acronym. It seems like an unnecessary layer and Decentralized docs is niche enough of a subject that most people won't know about it all the time. It seems okay in this doc, cause the acronym DD is defined at top. 
I'm mostly giving my opinion here, please feel free to use your judgement(a non-blocking comment)

Reply:
I 100% agree with you, but I was just fixing RST warnings here. Feel free to add a PR if you want to update this throughout and I'd approve.